test/test_helper_discovery.rb

```diff
@@ -1,3 +1,7 @@
+root = File.expand_path(File.dirname(__FILE__))
+require "#{root}/factories/discovery_rule_related"
+require "#{root}/factories/discovery_host_related"
```

Comment:
This adds possibility to run individual tests:

    be rake test:discovery TEST=/home/lzap/work/foreman_discovery/test/unit/discovered_extensions_test.rb